examples/substreams-powered-subgraph/schema.graphql

```diff
@@ -0,0 +1,5 @@
+type Contract @entity {
+  id: ID! #address of the contract
```

Comment:
The `#` notation is for adding comments to GraphQL files, but I think that in this case the content can also be a description? (and we can use `"` or `"""`). This way it will also be available in GraphiQL :)